python/frontend/cirrus/examples/test.py

```diff
@@ -0,0 +1,29 @@
+import time
+
```

Comment:
I would change the name of this file and function since "test" seems a bit vague.